csm_web/scheduler/models.py

```diff
@@ -1,25 +1,30 @@
 import datetime
-
 from django.db import models
 from django.conf import settings
 from django.contrib.auth.models import AbstractUser
-from django.core.exceptions import ValidationError
-from django.utils.translation import gettext_lazy as _
 from django.utils import timezone
+from rest_framework.serializers import ValidationError
 
 
 class User(AbstractUser):
     pass
 
 
-class ActivatableModel(models.Model):
-    active = models.BooleanField(default=True)
+class ValidatingModel(models.Model):
+    """
+    By default, Django models do not validate on save!
+    This abstract class fixes that insanity
+    """
```

Comment:
I thought this wasn't true, but then I checked the docs: https://docs.djangoproject.com/en/2.2/ref/models/instances/#django.db.models.Model.full_clean
just want to say, yikes